testservices/novaservice/service_test.go

```diff
@@ -252,29 +252,6 @@ func (s *NovaSuite) TestAddServerTwiceFails(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, `conflictingRequest: A server with id "test" already exists`)
 }
 
-// A control point can be used to change the status of the added server.
-func (s *NovaSuite) TestAddServerControlPoint(c *gc.C) {
```

Comment:
Why was this deleted? Add the reason in the description if it was a driveby tidyup.